lib/cn/cn.c

```diff
@@ -851,6 +858,11 @@ cn_open(
     struct mpool_props mpprops;
     struct merr_info   ei;
 
+    char kbuf[HSE_KVS_KEY_LEN_MAX];
```

Comment:
Get rid of those comments above cn_open().  I know they're driving you mad.  Lose the other "section" comments too.

Reply:
Now that you pointed them out I cannot unsee them!!!

Reply:
Done.